base/applications/rapps/loaddlg.cpp

```diff
@@ -798,10 +798,10 @@ CDownloadManager::ThreadFunc(LPVOID param)
         urlComponents.dwStructSize = sizeof(urlComponents);
 
         urlLength = InfoArray[iAppId].szUrl.GetLength();
-        urlComponents.dwSchemeLength = urlLength + 1;
+        urlComponents.dwSchemeLength = UINT(urlLength + 1);
```

Comment:
Use DWORD here and below

Reply:
It does not matter.

I thought I'd be nice and get rid of some annoying warnings but fuck it, I'll just back this out and let somebody else deal with it.